metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -120,29 +121,35 @@ final class BloopServers(
             shutdownServer()
             reconnect().ignoreValue
           case _ =>
-            Future.successful(())
+            Future.unit
         }
     } else {
-      Future.successful(())
+      Future.unit
     }
   }
 
   private def writeJVMPropertiesToBloopGlobalJsonFile(
       bloopGlobalJsonFilePath: AbsolutePath,
       bloopCreatedByMetalsFilePath: AbsolutePath,
-      requestedBloopJvmProperties: List[String],
+      maybeRequestedBloopJvmProperties: Option[List[String]],
       maybeJavaHome: Option[String]
   ): Try[Unit] = Try {
 
-    val javaOptionsString =
-      s"\"javaOptions\": [${requestedBloopJvmProperties.map(property => s"\"$property\"").mkString(", ")}]"
+    val javaOptionsString = maybeRequestedBloopJvmProperties
```

Comment:
Btw. if `maybeRequestedBloopJvmProperties` is empty we might want to use the existing options that are in the file.